compiler/bin-js_of_ocaml/compile.ml

```diff
@@ -65,6 +66,14 @@ let run
   | `Name _, _ -> ());
   List.iter params ~f:(fun (s, v) -> Config.Param.set s v);
   List.iter static_env ~f:(fun (s, v) -> Eval.set_static_env s v);
+  Config.Flag.(
+    match target_env with
+    | `Nodejs | `Isomorphic -> enable "include-node"
+    | _ -> disable "include-node");
```

Comment:
Please use exhaustive pattern matching. We want the compiler to complain if we ever add another target.

Reply:
ya, good call

Reply:
37f5605180